solidity/contracts/TokenGrant.sol

```diff
@@ -222,22 +222,19 @@ contract TokenGrant {
     function receiveApproval(address _from, uint256 _amount, address _token, bytes memory _extraData) public {
         require(ERC20Burnable(_token) == token, "Token contract must be the same one linked to this contract.");
         require(_amount <= token.balanceOf(_from), "Sender must have enough amount.");
-        require(_extraData.length == 137, "Invalid extra data length.");
```

Comment:
Having #1592 in `master` we shouldn't see this change here if we merge in the recent `master` here.

Reply:
I merged it into `managed-grant`, not `master`

Reply:
Ah, I missed this one. Cool.